servermanagement/AddWOtoVENV.sh

```diff
@@ -85,7 +85,17 @@ cd ./wo_highlight_dashboard
 "$PYTHON_CMD" setup.py develop
 cd ..
 
+echo -e "\n--- Installing wo_bulk_essay_analysis. ---"
+cd ./wo_bulk_essay_analysis
+"$PYTHON_CMD" setup.py develop
+cd ..
 
+echo -e "\n--- Installing wo_common_student_errors. ---"
+cd ./wo_common_student_errors
+"$PYTHON_CMD" setup.py develop
+cd ..
 
-
-
+echo -e "\n--- Installing wo_document_list. ---"
```

Comment:
The Writing Observer document list dashboard is a prototype dashboard exploring different ways of fetching docs. This is **not** meant to be teacher facing at this point in time, so this should **not** be installed on the any deployed server for the teachers.

This experimental dashboard will eventually be integrated into its own component/workflow for use with other dashboards.